packages/core/cms/faststore/content-types.json

```diff
@@ -271,6 +271,46 @@
                       "default": "Receive Tomorrow"
                     }
                   }
+                },
+                "tags": {
+                  "title": "Delivery Promise tags",
+                  "type": "object",
+                  "description": "The tag specified in this section will be displayed in product cards. Only one tag can be displayed.",
+                  "properties": {
+                    "enabled": {
+                      "title": "Should enable tags?",
+                      "type": "boolean",
+                      "default": false
+                    },
+                    "deliveryOptionTag": {
+                      "title": "Delivery Option tag",
+                      "type": "object",
+                      "properties": {
+                        "enabled": {
+                          "title": "Should enable Delivery Option tag?",
+                          "type": "boolean",
+                          "default": false
+                        },
+                        "deliveryOptionId": {
+                          "title": "Delivery Option ID",
+                          "type": "string",
+                          "default": "",
+                          "description": "The unique ID that represents the desired Delivery Option to be displayed as tag on product cards."
+                        }
```

Comment:
We need to confirm if we gonna still use it since IS will send the `name` property to be used as label.

Reply:
The ID in hCMS we need anyway so we know which DO we should use as tag. IS will return all DOs related to the product, but we'll show only 1 as the tag.